java/test_flatbuf1/Test.java

```diff
@@ -57,11 +57,11 @@ private String printNI(NIReader v) {
     }
 
     private String printQI(QIReader v) {
-        Map<Long,String> vs = v.m();
+        List<Tuple2__bit_32___stringReader> vs = v.m();
```

Comment:
this is an API change.
I assume lists are more efficient, but this is somewhat unintuitive. You'd think that a map in DL maps to a map in Java.
Also, the usability of the Map is better, since you don't have to figure out this Tuple name.

Reply:
I guess you are doing this because you represent collections by iterating over their elements.

Reply:
The compiler simply does not know that something behaves like a map any longer. We can add another attribute for this or just live with this and let the client convert between maps and arrays of tuples themselves.